storage/innobase/include/row0quiesce.h

```diff
@@ -46,6 +46,7 @@ struct trx_t;
 /** The v3 .cfg writes instant column default values in metadata section. */
 #define IB_EXPORT_CFG_VERSION_V3 0x3UL
 #define IB_EXPORT_CFG_VERSION_V1_WITH_RK 0xFFFFFFFF
+#define IB_EXPORT_CFG_VERSION_V3_WITH_RK 0xFFFFFFFD
```

Comment:
RK should be avoided. I see that there is hangover (zsolt also uses this at some places). If it is at only few places in the row0queisce* files, please fix them as part of this PR.

If there are many, please create separate PR for removal of RK in entire code base.

Reply:
Changed and created PS-5862: Make sure that rotation key is not mentioned in code base.